R/sanitize_maf_data.R

```diff
@@ -47,9 +48,18 @@ sanitize_maf_data = function(mutation_maf_path,
     warning("you should provide a list of genes to retain in the output to ensure your genes of interest are included")
     genes_keep = rev(unique(lymphoma_genes$Gene))
   }
-  maf_o = maftools::read.maf(mutation_maf_data)
-
-  maftools::oncoplot(maf_o, genes = genes_keep, writeMatrix = T, removeNonMutated = F)  #writes to working directory
+  maf_o = as.data.frame(mutation_maf_data)
+  onco_matrix <- create_onco_matrix(
+    maf_df = maf_o,
+    genes = genes_keep
+  ) %>% as.data.frame
+  #writes to working directory
+  write.table(
+    om,
```

Comment:
```
> safe_oncomatrix_path = sanitize_maf_data(
+  mutation_maf_data = GAMBLR.data::sample_data$grch37$maf,
+  genes_keep = c("MYC", "ID3", "ARID1A", "FOXO1", "TP53", "FAT4", "IGLL5")
+ )
There are no missing genes during onco matrix assembly.
Error in write.table(om, file = paste0(getwd(), "/onco_matrix.txt"), quote = F,  : 
  object 'om' not found
```

Reply:
Also, can you indent the examples?

Reply:
These both are fixed in the latest commit :+1:

Reply:
After running the function, I got this:
```
There are no missing genes during onco matrix assembly.
your data is in: /home/vsouza/repos/gamblr_repos/gambl_results_new_test_exp_data/onco_matrix.tsv
```
However, the the extension of the created file is txt. Could you change `tsv` to `txt` in the output message and in the returned string?

Reply:
Done! :heavy_check_mark: